pkg/agent/openflow/network_policy.go

```diff
@@ -991,6 +999,34 @@ func (c *client) InstallPolicyRuleFlows(rule *types.PolicyRule) error {
 	return nil
 }
 
+func (f *featureNetworkPolicy) isIGMPEgressRule(tableID uint8, isIngress, isIGMP bool) bool {
+	if !f.enableMulticast || isIngress || !isIGMP {
+		return false
+	}
+	return true
+}
+
+func (f *featureNetworkPolicy) isIGMPIngressRule(tableID uint8, isIngress, isIGMP bool) bool {
+	if !f.enableMulticast || !isIngress || !isIGMP {
+		return false
+	}
+	return tableID == MulticastIGMPIngressTable.GetID()
+}
+
+func (f *featureNetworkPolicy) isMulticastEgressRule(tableID uint8, isIngress, isIGMP bool) bool {
```

Comment:
Please add comments to explain what this func does.

Reply:
done